fastlane/metadata/android/cs-CZ/changelogs/103.1.txt

```diff
@@ -0,0 +1,9 @@
+- Přidány nové ikony s hladkým škálováním
+- Přidána černá díra na kapaliny (přispěno uživatelem @GioIacca9)
+- Přidáno nastavení průsvitnosti přemostění přepravníku (přispěno uživatelem @Quezler)
+- Přidána brána s podtokem (protikus k bráně s přetečením)
+- Přidány emotikony do kanálu zpráv, pro většinu bloků a předmětů
+- Přidán nový strom technologií, s lepší podporou pro modifikace
+- Přidán soubor s logem hry, uložený v adresáři s daty
+- Přidán nová oddělovat pro sprity a animace (ocená modéři)
```

Comment:
Tady je to nějaké rozbité. :) 

Nová nebo nový oddělovač? 

Ocená modéři =>  Ocení modéři.

Reply:
díky